src/gei/Commands/ReclaimMannequinCommand.cs

```diff
@@ -0,0 +1,132 @@
+            {
+                IsRequired = false
+            };
+
+            AddOption(githubTargetOrgOption);
+            AddOption(mannequinUsernameOption);
+            AddOption(targetUsernameOption);
+            AddOption(forceOption);
+            AddOption(verbose);
+
+            Handler = CommandHandler.Create<string, string, string, bool, bool>(Invoke);
+        }
+
+        public async Task<bool> Invoke(
+          string githubTargetOrg,
+          string mannequinUser,
+          string targetUser,
+          bool force = false,
+          bool verbose = false)
+        {
+            const string targetApiUrl = "https://api.github.com";
+
+            _log.Verbose = verbose;
+
+            _log.LogInformation("Reclaming Mannequin...");
+
+            _log.LogInformation($"GITHUB TARGET ORG: {githubTargetOrg}");
+            _log.LogInformation($"Mannequin: {mannequinUser}");
+            _log.LogInformation($"Reclaiming User: {targetUser}");
+
+            var githubApi = _targetGithubApiFactory.Create(targetApiUrl);
+            var githubOrgId = await GetOrgId(githubApi, githubTargetOrg);
+
+            var mannequin = await githubApi.GetMannequin(githubOrgId, mannequinUser);
+
+            if (mannequin == null || mannequin.Id == null)
+            {
+                _log.LogError($"User {mannequinUser} is not a mannequin.");
+                return false;
+            }
+            else if (mannequin.MappedUser != null && force == false)
+            {
+                _log.LogError($"User {mannequinUser} has been already mapped to {mannequin.MappedUser.Login}. Use the force option if you want to reclaim the mannequin again.");
+                return false;
+            }
+
+            var targetUserId = await githubApi.GetUserId(targetUser);
+
+            if (targetUserId == null)
+            {
+                _log.LogError($"Target user {targetUser} not found.");
+                return false;
+            }
+
+            var reclaimed = await githubApi.ReclaimMannequin(githubOrgId, mannequin.Id, targetUserId);
+
+            if (reclaimed)
+            {
+                _log.LogInformation($"{mannequinUser} ({mannequin.Id}) mapped to {targetUser} ({targetUserId})");
+            }
+            else
+            {
+                _log.LogInformation($"Failed to map {mannequinUser} ({mannequin.Id}) to {targetUser} ({targetUserId})");
+            }
+
+            return reclaimed;
+        }
+
+        public async Task<string> GetOrgId(GithubApi github, string githubOrg)
+        {
+            if (!string.IsNullOrWhiteSpace(githubOrg) && github != null)
+            {
+                _log.LogInformation($"GITHUB ORG: {githubOrg}");
+                var orgId = await github.GetOrganizationId(githubOrg);
+
+                _log.LogInformation($"    Organization Id: {orgId}");
+
+                return orgId;
+            }
+
+            throw new ArgumentException("All arguments must be non-null");
+        }
```

Comment:
This pattern is a bit weird, usually the exceptional checks happen in the beginning of a method and then if everything is OK it continues. so I'd re-write it as below:
```c#
if (githubOrg.IsNullOrWhiteSpace())
{
    throw new ArgumentNullException(nameof(githubOrg));
}
if (github == null)
{
    throw new ArgumentNullException(nameof(github));
}

... 
```

Reply:
I followed the pattern already being used in other parts of the code [eg](https://github.com/tspascoal/gh-gei/blob/631ba2eaf1f94a8aed683e1b8f5efe9e1bf50a1e/src/gei/Commands/GenerateScriptCommand.cs#L227) happy to change, but was just being consistent

Reply:
Well that example is also weird too 😞  we also need to probably refactor it later. But it certainly doesn't mean that we should continue doing it since it's already been done in our codebase 😃

Reply:
updated [3ef7445](https://github.com/github/gh-gei/pull/271/commits/3ef7445ef2948a4365aa2a59a309115af703eb7b)